addons/l10n_pe/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, api
+
+
+class ResCompany(models.Model):
+
+    _inherit = "res.company"
+
+    @api.onchange('country_id')
+    def onchange_country(self):
+        """In Peru, the rounding method it's calculated as global"""
+        for rec in self.filtered(lambda x: x.country_id == self.env.ref('base.pe')):
```

Comment:
should be done on CoA install (same was done for cl/ar)

Reply:
Complimented with the _load technique used on cl/ar

Reply:
done